src/pages/home/HomePage.tsx

```diff
@@ -28,9 +29,9 @@ const HomePage = () => {
                   flexDirection="column"
                   justifyContent="space-between"
                 >
-                  <Typography variant="h5">{item.title}</Typography>
+                  <Typography variant="h5" id={`cardTitle-${item.title}`}>{item.title}</Typography>
                   <Box alignSelf="flex-end">
-                    <ArrowForward color="primary" />
+                    <ArrowForward id="iconArrow" color="primary" />
```

Comment:
@leleOFA anche questo: l'icona per andare avanti sarebbe ripetuta in più card

Reply:
fix it